pyproject.toml

```diff
@@ -79,6 +79,7 @@ hathorlib = "0.3.0"
 pydantic = "~1.10.13"
 pyyaml = "^6.0.1"
 typing-extensions = "~4.8.0"
+python-healthchecklib = {git = "https://github.com/hathornetwork/python-healthcheck-lib"}
```

Comment:
Should we pin a specific commit/release?

Reply:
Done in https://github.com/HathorNetwork/hathor-core/pull/826/commits/fb612398edbb9ece11b4ad190ae2512688306cad